lib/daisy_ui_components/loading.ex

```diff
@@ -0,0 +1,54 @@
+defmodule DaisyUIComponents.Loading do
```

Comment:
missing tests

Reply:
[testing and rest attr](https://github.com/phcurado/daisy_ui_components/pull/31/commits/aba655a6c7838976866ca8820c69e1832f266892)